docs/DeveloperGuide.adoc

```diff
@@ -279,104 +281,334 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
+A specialized Appeal Administrator that handles module appeals during
+the hectic start-of-semester module registration in the National
+University of Singapore. He/She:
+
+* prefers desktop apps over other types
 * prefers typing over mouse input
+* can type fast
 * is reasonably comfortable using CLI apps
+* has a need to view large lists of modules, appeal cases, and student
+particulars
+* has a need to edit current student/appeal/module information to
+resolve appeal cases
+* has *neither the authority nor need* to add/remove students, appeals,
+and modules to/from the current database (out of job purview)
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: View, manage, and resolve appeals as fast as or
+faster than a typical mouse/GUI driven app.
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
+Epic: As a admin, I want to be able to view module/student/appeal data.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a/an … |I want to … |So that I can…
+|`* * *` |admin |be able to view lists of students/appeals/modules |
+
+|`* * *` |admin |filter lists of students/appeals/modules by year/module
+info/type |quickly find specific items of interest
 
-|`* * *` |user |add a new person |
+|`* * *` |admin |be able to view the full expanded details of a
+student/appeal/module |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |admin |be able to view lists or items side-by-side |easily
+cross-reference information between modules/students/appeals.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |admin |be able to find a module/appeal/student using their
+unique ID sequences |locate details of persons without having to go
+through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |admin |check for clashes between modules |determine if a
+student can take up the module he/she requested for
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |admin |check for clashes in a student’s timetable |determine
```

Comment:
Sounds more like a non functional requirement, since having clashes would make the product "unfunctional"?